test_apipkg.py

```diff
@@ -551,11 +551,25 @@ def test_aliasmodule_aliases_an_attribute():
     assert not hasattr(am, "lqkje")
 
 
-def test_aliasmodule_aliases_unimportable():
+def test_aliasmodule_aliases_unimportable_fails():
     am = apipkg.AliasModule("mymod", "qlwkejqlwe", "main")
     r = repr(am)
     assert "<AliasModule 'mymod' for 'qlwkejqlwe.main'>" == r
-    assert am.qwe is None
+    # this would pass starting with apipkg 1.3 to work around a pytest bug
```

Comment:
Which bug? Do we have an issue to link? (the answers could be in a comment in the code)

Reply:
#10